exercises/02-publish-subscribe/instructions.md

```diff
@@ -95,17 +95,10 @@ The `OrderSubmittedHandler` should process the `OrderSubmittedEvent` published b
 
 ### Step 3
 
-In the `App.config` file in the `Divergent.Shipping` project, add a new configuration section called `UnicastBusConfig` and specify `MessageEndpointMappings` for the assembly containing `OrderSubmittedEvent` event. The route information provided in the mapping is used by NServiceBus internally to send subscription requests from the subscriber to the publisher.
-
-```xml
-<configSections>
-  <section name="UnicastBusConfig" type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core" />
-</configSections>
-<UnicastBusConfig>
-  <MessageEndpointMappings>
-    <add Assembly="Divergent.Sales.Messages" Endpoint="Divergent.Sales" />
-  </MessageEndpointMappings>
-</UnicastBusConfig>
+In the `Divergent.Shipping` project, configure the publisher for the `OrderSubmittedEvent`. To do this use the `routing` object obtained when configuring the transport and add the following statement
```

Comment:
for the `OrderSubmittedEvent` event.

I sounds a bit odd, but then it should not include the word "the". Maybe we should remove the "Event" suffix and explain that events are in past tense? Or create another PR for that ;-)